.github/workflows/integration-deployment.yml

```
@@ -0,0 +1,76 @@
+name: J1 Google Cloud Integration deployment
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  j1-integration-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x        
+      - name: Get graph project, integration name and version number
+        id: set_constants
+        uses: actions/github-script@0.9.0
+        with:
+          script: |
+            const tagName = context.payload.release.tag_name
+            const versionNumber = tagName.replace("v", "")
+            core.setOutput('versionNumber', versionNumber)
+            core.setOutPut('integrationName', 'integration-google-cloud')
+            core.setOutPut('graphProjectName', 'google-cloud')
```

```suggestion
            core.setOutput('integrationName', 'integration-google-cloud')
            core.setOutput('graphProjectName', 'google-cloud')
```
Or to avoid using the deprecated core.setOutput more I would recommend removing these 2 lines and on line 8 using
```yml
jobs:
  j1-integration-deployment:
    runs-on: ubuntu-latest
    env:
      IntegrationName: integration-google-cloud
      GraphProjectName: google-cloud
```
and you can refer to them with `$integrationName` and `$GraphProjectName`

---

thanks @Nick-NCSU , committing suggested changes.